Tests/Clappr_Tests/Classes/Base/Layers/MediaControlLayerTests.swift

```diff
@@ -0,0 +1,39 @@
+import Quick
+import Nimble
+@testable import Clappr
+
+class MediaControlLayerTests: QuickSpec {
+    override func spec() {
+        describe(".MediaControlLayer") {
+            context("When a MediaControl is attached") {
+                it("resizes to match the layer bounds") {
+                    let mediaControlView = UIView(frame: .zero)
+                    let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+                    let layer = MediaControlLayer(frame: frame)
+
+                    layer.attachMediaControl(mediaControlView)
+                    layer.layoutIfNeeded()
```

Comment:
Do we need this call? If we do, could we encapsulate it inside the attach method itself so it's contained?

Reply:
This call is not needed, as the implementation code already calls it.

Reply:
Yes, we do, but it not necessary to encapsulate this inside the attach method because the `LayerComposer` already handles this when we call the `compose`. Since we are testing only the `MediaControlLayer` we need to explicitly call  `layoutIfNeeded`

Reply:
We changed the media control layer to update its parent view layout instead of MediaControlView (child view) on `attachMediaControl(_ mediaControl: UIView)`